adapter-http/src/main/resources/activities/documents-api/http-docsapi-crud-dataset.yaml

```diff
@@ -22,9 +23,11 @@ bindings:
   weighted_hosts: WeightedStrings('<<docsapi_host:stargate>>')
   # http request id
   request_id: ToHashedUUID(); ToString();
+  request_token: Discard(); TextOfFile(TEMPLATE(stargate_tokenfile,'data/stargate_token.txt'));
 
   seq_key: Mod(<<docscount:10000000>>); ToString() -> String
   random_key: Uniform(0,<<docscount:10000000>>); ToString() -> String
+  document_json: Discard(); TextOfFile('data/docsapi-dataset.json', false));
```

Comment:
I would prefer to see us split this out to a dedicated function, like `File(...)`. Booleans as signatures are not direct in their meaning, and often confuse users.

Reply:
Right, if this is the case to include File(...), we may need to consider the existing `TextOfFile` to be something more specific by name (e.g. LineOfFile(...) to indicated that only one line is being returned.  This will require some refactoring throughout its current usage in the scenarios and tests but when I read TextOfFile I'm thinking basically what we would now be introducing and intending with File(...).  wdyt?

Reply:
Changed to FirstLines(<path>, <number>) where no number specified indicates 1 line.